src/repositories/jobs.js

```diff
@@ -14,9 +15,63 @@ const create = async (payload) => {
   }
 };
 
+const createWithRecruiter = async (payload) => {
+  try {
+    let userId = payload.user_uuid;
+    let recruiterName = payload.recruiter_name;
+    let recruiterDescription = payload.recruiter_description;
+    delete payload.recruiter_description;
+    delete payload.recruiter_name
+    delete payload.user_uuid
```

Comment:
@rohan099 I think we can clean up this logic, by creating a new object instead of deleting the keys in the existing payload object.

Think of it in this way, in the future in case you need to pass more keys to the payload which we don't need to save in the database, going with current logic you'll need to delete all those keys, moreover, any change in the prop name will also break this logic.

So instead you can create a new object in this function where you just pick the keys required to be saved in the database.